src/v/kafka/server/handlers/fetch.cc

```diff
@@ -790,6 +791,9 @@ class simple_fetch_planner final : public fetch_planner::impl {
                 bytes_left_in_plan -= max_bytes;
             }
 
+            const auto client_address = fmt::format(
+              "{}", octx.rctx.connection()->client_host());
```

Comment:
Adding the address seems non-controversial to me.

That said, I'm wondering if address is unique enough of an identifier to be useful in logs. I wonder if we should consider having an actual fetch identifier (eg some counter per shard)

Reply:
I had missed adding the port earlier, added it now. There was a discussion around using a UUID or the correlation id here, but the logs could become quite verbose with a UUID. An increasing counter does seem more useful.

Reply:
FWIW, depending on how things are wired up in the infrastructure, there will be cases where the only client IP address Redpanda may get is that of a load balancer. For example, this case is very likely when accessing Redpanda through cloud private links.

Reply:
Thanks for pointing this out. This is suboptimal but acceptable for now, the idea behind this change set is to add more information where/if possible. 

It should drive us to use a better identifier eventually, something like @andrwng mentioned.